edgekv/lib/readme.md

```diff
@@ -113,6 +135,15 @@ The edgekv.js file must be included alongside `edgekv_tokens.js` which includes
 	} catch (error) {
 		// do something in case of an error
 	}
+### Write the Product Modification Time Without Wait For Response
```

Comment:
Mention in the title that this is an example, or possibly put under an examples section?

Reply:
It is in the examples section.

Reply:
my bad, the PR is not showing whole file for some reason. I'll merge it in.